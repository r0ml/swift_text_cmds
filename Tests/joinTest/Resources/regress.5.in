Belgium,Brussels
France,Paris
Italy,Rome
Switzerland
