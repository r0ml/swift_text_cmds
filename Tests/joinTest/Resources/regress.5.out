Permanent International Peace Bureau,1910,
